configs/experiments/coco_ddpn.yaml

```diff
@@ -0,0 +1,17 @@
+exp:
+  name: coco_people_ddpn
+  log_dir: logs
+model:
+  weights: weights/coco_people_ddpn/state_dict_best_loss.ckpt
+  test_cfg: configs/test/coco_ddpn_cfg.yaml
+data:
+  test_examples: Null
+  n_samples: 1
+  perturb: Null
+  module: coco_people # {ood_blur_coco_people, ood_mixup_coco_people, coco_people}
+hyperparams:
+  lmbda: 0.01
```

Comment:
I'm seeing a few different lmbda values across these configs. I think we said $\lambda = 0.1$ everywhere in the paper. Is this a leftover setting?

Reply:
Leftover from last minute hyperparam search. All defaults are now 0.1